ui/src/components/easter/hero/HeroContainer.js

```diff
@@ -1,105 +1,94 @@
 import 'react-modal-video/scss/modal-video.scss';
-import React, { Fragment, useState } from 'react';
+import React,{Fragment, useState} from "react";
```

Comment:
in general, please keep your code tidy (remove unused import, check formatting, indents, etc. (look into prettier to automate all of this). Also, if you could combine your 3 commits into 1, just looks cleaner that way